### DISCUSSION TITLE
Ergo - Workers getting disconnected

### DISCUSSION THREAD

**Discussion by jvxis** [coinfoundry/miningcore#1393] [General]
My workers are getting disconnected time from time. No specific reason. I'm testing with T-REX and Nbminer. Same behavior. All of them disconnected at the same time.
I ran in debug mode, and I received the message:
[2022-09-03 13:15:52.5291] [D] [ergo1] [0HMKDKAME1UR6] Received EOF
[2022-09-03 13:15:52.5337] [I] [ergo1] [0HMKDKAME1UR6] Connection closed
What the message "Received EOF" means?
Thanks
J

**Comment by ei8ht187** [comment]
Have the same problem but only when I connect via a loadbalancer with activated TcpProxyProtocol. No problems at all when the miner connects directly to the stratum, except for TRM. This doesn't work at all.

**Comment by oliverw** [comment]
> My workers are getting disconnected time from time. No specific reason. I'm testing with T-REX and Nbminer. Same behavior. All of them disconnected at the same time. I ran in debug mode, and I received the message: [2022-09-03 13:15:52.5291] [D] [ergo1] [0HMKDKAME1UR6] Received EOF [2022-09-03 13:15:52.5337] [I] [ergo1] [0HMKDKAME1UR6] Connection closed What the message "Received EOF" means? Thanks J

EOF means "End of File" or the remote peer (the miner) closed the connection. Not the pool.
